fix: formatting in response like

gabygabylagaby opened the pull request:
## **Trello Ticket**

[ID-10](https://trello.com/c/CSuNCXxB/12-id-10-comment-management)

---

## What did I do?

I implemented support for adding, editing, deleting, and retrieving comments on posts. This functionality allows users to interact with each other by commenting on posts and managing those comments.

- Created new REST endpoints in the `CommentsController` to manage comments.
- Implemented the necessary logic to add, edit, and delete comments in the database using the `CommentsRepository`.
- Created input request models for creating and editing comments (`CreateCommentRequest` and `EditCommentRequest`).
- Added validations to ensure that only valid data is submitted.
- Refactored the `CommentsService` to handle business logic and interact with the repository efficiently.
- Mapped responses using `CommentResponse` to ensure consistent data output.


---

## Why did I do this?
This functionality was needed to allow users to engage with posts by leaving comments, updating or deleting them as necessary.

- **Problem:** Users needed the ability to comment on posts and manage those comments (e.g., editing or deleting them) in order to better interact with each other.
- **Value added:** By enabling comment management, users can have richer social interactions on the platform, improving user engagement and content dynamism.

---

## How did I do it?
- Defined new endpoints:
  - `POST /posts/{postId}/comments` to add a comment.
  - `PUT /posts/{postId}/comments/{commentId}` to edit a comment.
  - `DELETE /posts/{postId}/comments/{commentId}` to delete a comment.
  - `GET /posts/{postId}/comments` to retrieve all comments on a post.
  
- Implemented the `CommentsService` to handle the logic of adding, updating, deleting, and retrieving comments.
- Developed the `CommentsRepository` to manage database persistence for comments.
- Utilized `Entity Framework` to handle data interaction with the database.
- Implemented `CreateCommentRequest` and `EditCommentRequest` models to validate input data.
- Mapped comment data using `CommentResponse` for consistent response output.


---

## (Optional) Notes:

jhael-arce-jalafund reviewed (changes requested):
The following endpoint defined in the wiki is missing![image](https://github.com/user-attachments/assets/c42519ae-f885-40e4-a522-bc480ea78ceb)

jhael-arce-jalafund reviewed (changes requested):
delete the following files
![image](https://github.com/user-attachments/assets/a4942946-d435-4db5-a188-fc772fd326c4)

jhael-arce-jalafund reviewed (changes requested):
user endpoints do not follow the format specified in the wiki
it should be /api/v1/users/{userId}
![image](https://github.com/user-attachments/assets/ecf7e9e6-0145-47f5-accc-e760c32bc4e8)

gabygabylagaby commented:
I think the last endpoint was not understood, which is that it gets the likes of a user's post:
![image](https://github.com/user-attachments/assets/8510ee79-513c-4f16-b90c-c6143504e202)
but I already deleted it as it is in the documentation and I already corrected it

jhael-arce-jalafund commented on SocialNetworkApi/Controllers/UserController.cs:
```diff
@@ -26,42 +26,42 @@ public async Task<IActionResult> CreateUser([FromBody] CreateUserRequest request
-        {
-            var users = await _userService.GetAllAsync(); 
-            var userResponses = users.Select(UserResponse.FromDomain); 
+    [HttpGet]
+    [Route("allusers")]
+    public async Task<IActionResult> GetAllUsers()
+    {
+        var users = await _userService.GetAllAsync();
+        var userResponses = users.Select(UserResponse.FromDomain);
 
         return Ok(userResponses);
     }
```
The name allusers could be improved to users, getAllUsers, or listUsers to make it more descriptive and in line with REST standards.

jhael-arce-jalafund reviewed (changes requested):
Let a non-existent user like a post
![image](https://github.com/user-attachments/assets/adc366e9-00ec-4fc0-b03e-dfdf4a418786)

jhael-arce-jalafund reviewed (changes requested):
A user should only be able to place 1 like, but can place many
![image](https://github.com/user-attachments/assets/1c9cfb54-8b2f-4292-9e1e-8cb55d91b008)

gabygabylagaby commented:
@jhael-arce-jalafund these validations are another US

jhael-arce-jalafund reviewed (changes requested):
I should be able to see all the likes that the user made, but it tells me that  did not make any likes.
![image](https://github.com/user-attachments/assets/514d4ae7-392f-4010-ae13-08f4c68b2b45)

gabygabylagaby commented:
@jhael-arce-jalafund Done

jhael-arce-jalafund reviewed (approved):
Good Job!
